index.js

```diff
@@ -14,11 +14,17 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell
-//. and PureScript. It's stricter and more opinionated than [Ramda][],
-//. and provides a similar suite of functions.
+//. Sanctuary is a functional programming library inspired by Haskell and
+//. PureScript. It provides a similar suite of functions as [Ramda][],
+//. but in a stricter and more opinionated way, ensuring errors are
+//. properly handled.
+//. Sanctuary makes it easy to build a program up from smaller types and
+//. functions and makes it possible to write safe code without constant
+//. null-checks.
```

Comment:
I'd like to see the word "composition" in the paragraph above. How about something like this?

_Sanctuary promotes programs composed of simple, pure functions. Such programs are easy to test, comprehend, and maintain, and a pleasure to write._

_Sanctuary provides two [Fantasy Land][1] -compatible data types, [Maybe][2] and [Either][3], which mean that even functions which may fail—such as [`head`][4]—are composable._


[1]: https://github.com/fantasyland/fantasy-land
[2]: https://sanctuary.js.org/#maybe-type
[3]: https://sanctuary.js.org/#either-type
[4]: https://sanctuary.js.org/#head